app/boards/shields/kyria/keymaps/default/keymap.overlay

```diff
@@ -0,0 +1,50 @@
+#include <dt-bindings/zmk/keys.h>
+#include <dt-bindings/zmk/keymap-transform.h>
+#include <keymap.h>
+
+/ {
+	chosen {
+		zmk,keymap = &keymap0;
+	};
+	
+	keymap0: keymap {
+		compatible = "zmk,keymap";
+		label ="Default Kyria Keymap";
+		layers = <&default>;
+		transform = <&default_transform>;
+	};
+
+	default_transform: keymap_transform_0 {
+		compatible = "zmk,keymap-transform";
+		map = <
+			IDENTITY_ROW(0, 6)       OFFSET_ROW(6, 0, 6)
+			IDENTITY_ROW(1, 6)       OFFSET_ROW(6, 1, 6)
+			IDENTITY_ROW(2, 8)       OFFSET_ROW(8, 2, 8)
+			OFFSET_ROW(3, 3, 5)      OFFSET_ROW(8, 3, 5)
+		>;
+	};
+
+	five_column_transform: keymap_transform_1 {
+		compatible = "zmk,keymap-transform";
+		map = <
+			IDENTITY_ROW(0, 5)       OFFSET_ROW(5, 0, 5)
+			IDENTITY_ROW(1, 5)       OFFSET_ROW(6, 1, 5)
+			IDENTITY_ROW(2, 7)       OFFSET_ROW(7, 2, 7)
+			OFFSET_ROW(2, 3, 5)      OFFSET_ROW(7, 3, 5)
+		>;
+	};
+
+	layers {
+		compatible = "zmk,layers";
+
+		default: layer_0 {
+			label = "DEFAULT";
+			keys = <
+	KC_ESC KC_Q KC_W KC_E KC_R KC_T                                   KC_Y KC_U KC_I KC_O KC_P KC_PIPE
```

Comment:
You can start to see how we could make this visually easier to understand, like QMK keymaps. I will clean this up even more in this PR to show it more completely.

Reply:
Ok, pushed an update here that shows the "logical layout" via comments for both the *transform* (shows the MX## pad details), and the keymap layers.

Reply:
Marking this resolved, after discussion on Discord.